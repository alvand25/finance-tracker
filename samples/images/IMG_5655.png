version https://git-lfs.github.com/spec/v1
oid sha256:b7a48abae79c886a65e6d4c6b3ecf8fa1bab3abb55a49cb5bea2310a07315d19
size 12960999
