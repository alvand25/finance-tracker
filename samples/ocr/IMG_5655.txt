: Ne fi
' MART"
http://www. hmart .com
: 48-18 Northern B1vd
. Lona Is1and City, NY. 11101
TEL (718) 489-8580
Your Cashier was 5YLVIA KIH
i PRODUCE 3.36 F
j MILK PAN BREAD 5.95 F
: BEAN 5PROLIT5/120z 1,99 F
: ORG BEECH MU5HROCM 3.99 F
: MA5AMAN CURRY PA5T 1.99 F
GREEN CURRY PA5TE 1.99 F
H5 PICKLED RADI5H 1,49 F
1 @ 4/1,.99
LIME 0.50 F ;
1 6 4/1 99
LIME 0.50 F
MRN BF RIBEYE ROLL 8.90 F
i Regu1ar Price 9.99 , You saved 0.99
RAMA FRE5H RICE 5T 2.99 F \
F2 BF TOP BLADE 16.79 F
TAX 0.00
sitet BALANCE 60.44
HasterCard Credit - H
ACCOUNT NUMBER: #e###x HHH 2835
fiPPROVAL CONE: 08458P
5EQUENCE NUMBER: 6717
TOTAL AAGUAT: 50,44
RE5PON5E CODE: APPROVED
04714725 04. 26en 128 6
MasterCard 50.44
CPE 0.00
core) 1 MtER Ce TTEN5 5OLD = 12