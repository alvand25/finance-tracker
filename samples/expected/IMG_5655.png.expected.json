{
  "store": "Costco",
  "items": [
    {
      "description": "KIRKLAND PAPER TOWELS",
      "price": 19.99
    },
    {
      "description": "BANANAS ORGANIC",
      "price": 5.99
    },
    {
      "description": "MILK 2% 3-PACK",
      "price": 8.99
    }
  ],
  "subtotal": 34.97,
  "tax": 2.80,
  "total": 37.77,
  "payment_method": "VISA",
  "timestamp": "2024-04-10T14:32:00",
  "check_items_exactly": true
}